src-shared/db-enum.js

```diff
@@ -74,6 +74,11 @@ exports.storageOption = {
   external: 'External',
 }
 
+exports.composition = {
+  fullFamily: 'fullFamily',
```

Comment:
Should an xml example for fullFamily be added as well?

Reply:
Sure but it won't add more test coverage